query_execution/tests/QueryManagerSingleNode_unittest.cpp

```diff
@@ -730,122 +731,4 @@ TEST_F(QueryManagerTest, TwoNodesDAGPartiallyFilledBlocksTest) {
   EXPECT_TRUE(getOperatorFinishedStatus(id2));
 }
 
-TEST_F(QueryManagerTest, MultipleNodesNoOutputTest) {
```

Comment:
This is not a far fetched scenario. Steps to reproduce: 

1. CREATE TABLE r (a INT, b INT);
2. CREATE TABLE s (c INT, d INT);
3. Select sum(a) from r, s where r.a = s.c;

Reply:
Good point!

Reply:
I've fixed this bug. Please take another look.

After that, I will squash both commits into one.